ddsrecorder_participants/src/cpp/recorder/mcap/McapSizeTracker.cpp

```diff
@@ -39,11 +39,11 @@ McapSizeTracker::~McapSizeTracker()
 }
 
 void McapSizeTracker::init(
-        const std::string& path,
-        const uint64_t& safety_margin)
+        const std::uint64_t& space_available,
+        const std::uint64_t& safety_margin)
 {
     logInfo(DDSRECORDER_MCAP_SIZE_TRACKER,
-            "Initializing tracker for: " << path);
+            "Initializing tracker with a size of " << space_available);
```

Comment:
The `McapSizeTracker` initializer no longer receives the path since the size estimation is done at a higher level. I believe it's more useful for the user to know the size available for recording than to see the path where he's recording.

Reply:
The `McapSizeTracker` initializer no longer receives the path since the size estimation is done at a higher level. I believe it's more useful for the user to know the size available for recording than to see the path where he's recording.